scripts/dotnet-cli-build/PublishTargets.cs

```diff
@@ -45,15 +49,63 @@ public static BuildTargetResult InitPublish(BuildTargetContext c)
         [Environment("PUBLISH_TO_AZURE_BLOB", "1", "true")] // This is set by CI systems
         public static BuildTargetResult Publish(BuildTargetContext c)
         {
+            if (ShouldPublishToLatest())
+            {
+                string targetContainer = $"{Channel}/Binaries/Latest/";
+
+                foreach (string blob in _latestBlobs)
+                    AzurePublisherTool.CopyBlob(blob.Replace("/dotnet/", ""), (targetContainer + Path.GetFileName(blob)).Replace(CliNuGetVersion, "latest").Replace(SharedFrameworkNugetVersion, "latest"));
+
+                PublishLatestCliVersionTextFile(c);
+                PublishLatestSharedFrameworkVersionTextFile(c);
```

Comment:
Does this mean we are going to publish the latest version text file only for the current RID?